### ISSUE TITLE
InvlidLoginAttempts recording in MongoDbAuthRepository

### ISSUE THREAD

**Issue by SteveHallNNT** [ServiceStack/Issues#328]
Hi
We're using the MongoDbAuthRepository and have noticed that the UserAuth.InvalidLoginAttempts is not being updated / cleared upon failed or successful logins.  The only use of this currently seems to be in the OrmLiteAuthRepository.  It seems a simple enough fix - just copy across the RecordInvalidLoginAttempt() and RecordSuccessfulLogin() functions and call them accordingly.
Many Thanks
Steve H

**Comment by mythz** [comment]
Hi Steve, have added support for InvalidLoginAttempts to all Auth Repositories [in this commit](https://github.com/ServiceStack/ServiceStack/commit/23940385d5d9deec07a9e2a23859e0c4c856ad2b)

To enable you'll need to set the `MaxLoginAttempts` option on the AuthFeature plugin, e.g:

```csharp
Plugins.Add(new AuthFeature(...) {
    MaxLoginAttempts = 5
});
```

This change is now available from v4.0.43+ that's now [available on MyGet](https://github.com/ServiceStack/ServiceStack/wiki/MyGet).

**Comment by SteveHallNNT** [comment]
That's great, thanks.
